python/cudf/cudf/core/dataframe.py

```diff
@@ -3161,24 +3161,29 @@ def from_pandas(cls, dataframe, nan_as_null=True):
 
         df = cls()
         # Set columns
-        for i, colk in enumerate(dataframe.columns):
+        for colk in dataframe.columns:
```

Comment:
Instead of iterating by names and then doing lookups we can directly iterate over the columns.
```suggestion
        for col_name, col_value in dataframe.iteritems():
```